sru-templates/manual/azure-sru

```diff
@@ -39,56 +71,76 @@ for series in $SRU_SERIES; do
         xenial) image_version=16.04-DAILY-LTS
                 netcfg="/etc/network/interfaces.d/50-cloud-init.cfg";;
         bionic) image_version=18.04-DAILY-LTS;;
-        cosmic) image_version=18.10-DAILY;;
         disco) image_version=19.04-DAILY;;
         *) echo "!! UPDATE FAMILY CASE STATEMENT !!"; exit 1;;
     esac
-    name=test-sru-$series
-    az vm create --name=$name \
-        --image=Canonical:UbuntuServer:$image_version:latest \
-        --admin-username=root --admin-username=ubuntu \
-        -g $RESOURCE_GROUP --ssh-key-value $SSH_KEY \
-        $BOOT_DIAG \
-        --custom-data sethostname.yaml
-    IP=$(az vm list-ip-addresses --name $name | jq -r '.[] | .virtualMachine.network.publicIpAddresses[].ipAddress'| awk '{printf "ubuntu@%s", $1}')
+    # Note: accelerated network requires bigger size than our default DS1
+    for nics in "" "--nics $NIC_NAME-$series --size Standard_DS2_v2"; do
+        if [ "" = "$nics" ]; then
+            echo "Creating standard network vm"
+            name=test-sru-$series
+        else
+            echo "Creating advanced network vm"
+            name=test-sru-$series-advanced
+            az network public-ip create \
+                --name sruPublicIp-$series \
+                --resource-group $RESOURCE_GROUP
+
+            az network nic create \
+                --resource-group $RESOURCE_GROUP \
+                --name $NIC_NAME-$series \
+                --vnet-name $VNET_NAME \
+                --subnet sruSubnet \
+                --accelerated-networking true \
+                --public-ip-address sruPublicIp-$series \
+                --network-security-group $NETWORK_SECURITY_GROUP
+        fi
+        az vm create --name=$name \
+            --image=Canonical:UbuntuServer:$image_version:latest \
+            --admin-username=root --admin-username=ubuntu \
+            -g $RESOURCE_GROUP --ssh-key-value $SSH_KEY \
+            $BOOT_DIAG \
+            --custom-data sethostname.yaml $nics
+        IP=$(az vm list-ip-addresses --name $name | jq -r '.[] | .virtualMachine.network.publicIpAddresses[].ipAddress'| awk '{printf "ubuntu@%s", $1}')
 
-    ssh "${sshopts[@]}" $IP -- cloud-init status --wait --long
-    ssh "${sshopts[@]}" $IP -- dpkg-query --show cloud-init
-    ssh "${sshopts[@]}" $IP -- sudo cat /run/cloud-init/result.json
-    ssh "${sshopts[@]}" $IP -- grep Trace /var/log/cloud-init.log
-    ssh "${sshopts[@]}" $IP -- systemd-analyze
-    ssh "${sshopts[@]}" $IP -- systemd-analyze blame
-    ssh "${sshopts[@]}" $IP -- cloud-init analyze show
-    ssh "${sshopts[@]}" $IP -- cloud-init analyze blame
-    echo 'Networking config'
-    ssh "${sshopts[@]}" $IP -- cat $netcfg
+        ssh "${sshopts[@]}" $IP -- cloud-init status --wait --long
```

Comment:
Because we are now nested inside a second for loop (to handle basic vs advanced networks) we have to indent all the scp/ssh operations within the additional for loop;